### DISCUSSION TITLE
Add ALT description to a logo for revealjs

### DISCUSSION THREAD

**Discussion by massimovassalli** [quarto-dev/quarto-cli#10747] [Q&A] [answered]
### Description

Hi,
I am creating a revealjs presentation in quarto with a logo. 
---
title: "Mytitle"
format: revealjs
logo: mylogo.png
---

Do you know how to force the generated HTML code to contain an ALT tag with a description? This will help having an accessible PDF printed out.

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.
See <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us>.

**Reply by mcanouil** [reply]
Duplicate of:

- https://github.com/quarto-dev/quarto-cli/discussions/8592

**Comment by mcanouil** [comment, marked as answer]
I've opened an enhancement issue for `logo-alt` in `revealjs`:
- https://github.com/quarto-dev/quarto-cli/issues/10933
